src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,73 @@
-export const Pagination = () => {};
+import React from 'react';
+import classNames from 'classnames';
+
+interface Props {
+  total: number;
+  perPage: number;
+  currentPage?: number;
+  onPageChange: (value: number) => void;
+}
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage = 1,
+  onPageChange,
+}) => {
+  const perPageSelectors: number =
+    total % Number(perPage) ? Math.floor(total / perPage) + 1 : total / perPage;
+
+  return (
+    <>
+      <ul className="pagination">
+        <li
+          className={classNames('page-item', {
+            disabled: currentPage === 1,
+          })}
+        >
+          <a
+            data-cy="prevLink"
+            className="page-link"
+            href="#prev"
+            aria-disabled={currentPage === 1}
+            onClick={() => onPageChange(currentPage - 1)}
```

Comment:
The previous page link can potentially set the `currentPage` to 0, which is not a valid page number. You should add a check to ensure that `currentPage - 1` does not fall below 1.